### DISCUSSION TITLE
How do change or create traceID from another way ?

### DISCUSSION THREAD

**Discussion by Bjxzc** [apache/skywalking#8448] [Q&A]
Could I use kong "correlation ID" instead of traceID in the skywalking?

**Comment by wu-sheng** [comment]
That is a thing, LUA tracer or Kong tracer should do. I think there isn't a way for now.

**Reply by wu-sheng** [reply]
And some other people may want to use `requestId` rather than `correlation ID`. So the recommended way is to open trace ID generator to different implementations.
